README.md

```diff
@@ -49,6 +48,9 @@ pip install -e .
 
```

Comment:
Should we update this to be python=3.12 as well?

BTW, I was testing with a fresh install on a new machine, and somehow the environment still installed numpy 2, so probably we should keep version 1.26.3 hardcoded.

Reply:
Ignore the comment regarding numpy! I was testing the wrong version of the code (see comment below)